sdk/compiler/damlc/daml-stdlib-src/DA/Text.daml

```diff
@@ -270,6 +271,9 @@ isAlpha = isPred (\t -> t >= "a" && t <= "z" || t >= "A" && t <= "Z")
 isAlphaNum : Text -> Bool
 isAlphaNum = isPred (\t -> t >= "0" && t <= "9" || t >= "a" && t <= "z" || t >= "A" && t <= "Z")
 
+isHex : Text -> Bool
+isHex = isPred (\t -> t >= "0" && t <= "9" || t >= "a" && t <= "f" || t >= "A" && t <= "F")
```

Comment:
I'm not currently using this function, but have left it in as it might be useful for validating hex strings (even after we've introduced `ByteString` or similar)